src/Collections-Support-Tests/ValueLinkForTesting.class.st

```diff
@@ -0,0 +1,8 @@
+"
+I am a variant of `ValueLink` to show the behavior of `Object>>#~~>`.
+"
```

Comment:
Why do you need a new subclass without any new methods?

Reply:
The new subclass has been introduced to show that the expression `a ~~> b` creates a link (with `a` as value) of the same class of the link `b`, so no additional methods are necessary to test it. Of course, in real usage those subclasses would hold new methods.

Reply:
Could you add this to the class comment? Because a subclass without no new method IS strange

Reply:
Added in 464c813